README.md

```diff
@@ -8,6 +8,11 @@ easy-to-use client library for the [Google Ads API][].
 
 # Documentation for Users
 
+**For thorough documentation of all features and utilities, visit our
+[Developers Site ruby guide][].**
+
```

Comment:
Lines 12-13 duplicate line 14

Reply:
No, line 14 defines where the link goes. I'm using a different type of markdown link here than we've used in other places.